packages/components/table/tests/table.test.tsx

```diff
@@ -293,24 +293,24 @@ describe("<Tbody />", () => {
 
   test("checkbox can be clicked and toggled", async () => {
     const { user } = render(<Table columns={columns} data={data} />)
+    const checkbox = screen.getByLabelText("Select row")
     await waitFor(async () => {
-      const checkbox = screen.getByLabelText("Select row")
       expect(checkbox).not.toBeChecked()
       await user.click(checkbox)
```

Comment:
And while you're at it, could you please fix this too?

Reply:
@108yen 
Fixed. Please check if this addresses the issue:
https://github.com/yamada-ui/yamada-ui/pull/3029/commits/4a14a14332b41b045b6e1ef7fa7092844af9bfbd